test/permission.validation.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const metatests = require('metatests');
+const util = require('util');
+
+const {
+  setup,
+  prepareRemoteProvider,
+} = require('./fixtures/permission-validation/setup');
+
+const runTests = (test, ctx) => {
+  const testsDir = path.join(__dirname, 'permissions');
+  const readdir = util.promisify(fs.readdir);
+  return readdir(testsDir).then(testFiles => {
+    for (const testFile of testFiles) {
+      const testFilePath = path.join(testsDir, testFile);
+      const subtest = require(testFilePath);
+      subtest(test, ctx);
```

Comment:
```suggestion
      subtest(test.test(testFile), ctx);
```

And you can omit `permission.create: ` in test message and `test.endAfterSubtest`.